trestle/core/commands/author/profile.py

```diff
@@ -158,7 +129,7 @@ def generate_markdown(
             yaml_header: Dict to merge into the yaml header of the control markdown
             overwrite_header_values: Overwrite values in the markdown header but allow new items to be added
             sections_dict: Optional dict mapping section short names to long
-            required_sections: Optional comma-sep list of sections that get prompted for prose if not in the profile
+            required_sections: Optional list of sections that get prompted for prose if not in the profile
```

Comment:
no longer an optional list

Reply:
It is and should be optional.  Changed accordingly in typing.